.azure-pipelines/release-cli.yaml

```diff
@@ -708,7 +708,7 @@ stages:
                   displayName: "NuGet push"
                   inputs:
                     command: push
-                    packagesToPush: "$(artifactsDownloadLocation)/nuget/Microsoft.Graph.Beta.Cli.*.nupkg;$(artifactsDownloadLocation)/nuget/Microsoft.Graph.Beta.Cli.*.snupkg;"
+                    packagesToPush: "$(artifactsDownloadLocation)/nuget/Microsoft.Graph.Beta.Cli.*.nupkg;!$(artifactsDownloadLocation)/nuget/Microsoft.Graph.Beta.Cli.*.snupkg;"
```

Comment:
I'm curious to why you need to explicitly exclude the snupkg? the first glob already does not include it, why not simply remove the second glob?

Reply:
It's meant to be `symbols.nupkg`. I've updated it.